CHANGELOG.md

```diff
@@ -34,6 +34,7 @@
 
 * [ObjectServer] `SyncUserInfo` now also exposes a users metadata using `SyncUserInfo.getMetadata()`
 * Minor performance improvement when copy/insert objects into Realm.
+* [ObjectServer] Added support for partial sync (#5276).
```

Comment:
Make it really explicit that this is just a Tech Preview and the API _will_ change.